src/main/resources/public/scss/loginPage/login-login.scss

```diff
@@ -121,8 +122,13 @@
       }
 
       .or {
+        position: absolute;
```

Comment:
use position relative, and text-align: center for parent block. Line must be absolute position. Do not use transform for set position element (only for animation)